sklearn/metrics/_classification.py

```diff
@@ -789,6 +791,8 @@ def matthews_corrcoef(y_true, y_pred, *, sample_weight=None):
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights.
 
+    .. versionadded:: 0.18
```

Comment:
The `versionadded` directive should be indented as in line https://github.com/scikit-learn/scikit-learn/blob/ff1e295be29384205e0147bb126158c5514d05e2/sklearn/metrics/_classification.py#L230
Right now the directive is rendered in bold but the needed rendering is italic.

Reply:
Done.